data-reconciliation-app/test/data-samples/member0_data.csv

```diff
@@ -1,5 +1,5 @@
 lei,nace
-984500E1B2CA1D4EKG67,A01.1 
+984500E1B2CA1D4EKG67,A.01.1
```

Comment:
The difference in this record from the other members was just a blank space.
Added a easier-to-see difference to avoid misunderstanding